content/millennium/dstu2/structure-definition.md

```diff
@@ -0,0 +1,75 @@
+---
+title: StructureDefinition | DSTU 2 API
+---
+
+# StructureDefinition
+
+* TOC
+{:toc}
+
+## Overview
+
+The StructureDefinition resource describes a FHIR structure including data elements and their usage. Our current implementation uses this resource to define custom extensions.
+
+The following fields are returned if valued:
+
+* [Id](http://hl7.org/fhir/dstu2/resource-definitions.html#Resource.id){:target="_blank"}
+* [Publisher](http://hl7.org/fhir/DSTU2/structuredefinition-definitions.html#StructureDefinition.publisher){:target="_blank"}
+* [Snapshot](http://hl7.org/fhir/DSTU2/structuredefinition-definitions.html#StructureDefinition.snapshot){:target="_blank"}
+* [Date](http://hl7.org/fhir/DSTU2/structuredefinition-definitions.html#StructureDefinition.date){:target="_blank"}
+* [Fhir version](http://hl7.org/fhir/DSTU2/structuredefinition-definitions.html#StructureDefinition.fhirVersion){:target="_blank"}
+* [Constrained type](http://hl7.org/fhir/DSTU2/structuredefinition-definitions.html#StructureDefinition.constrainedType){:target="_blank"}
+* [Description](http://hl7.org/fhir/DSTU2/structuredefinition-definitions.html#StructureDefinition.description){:target="_blank"}
+* [Status](http://hl7.org/fhir/DSTU2/structuredefinition-definitions.html#StructureDefinition.status){:target="_blank"}
+* [Context type](http://hl7.org/fhir/DSTU2/structuredefinition-definitions.html#StructureDefinition.contextType){:target="_blank"}
+* [Context](http://hl7.org/fhir/DSTU2/structuredefinition-definitions.html#StructureDefinition.context){:target="_blank"}
+* [URL](http://hl7.org/fhir/DSTU2/structuredefinition-definitions.html#StructureDefinition.url){:target="_blank"}
+* [Kind](http://hl7.org/fhir/DSTU2/structuredefinition-definitions.html#StructureDefinition.kind){:target="_blank"}
+* [Differential](http://hl7.org/fhir/DSTU2/structuredefinition-definitions.html#StructureDefinition.differential){:target="_blank"}
+* [Abstract](http://hl7.org/fhir/DSTU2/structuredefinition-definitions.html#StructureDefinition.abstract){:target="_blank"}
+* [Base](http://hl7.org/fhir/DSTU2/structuredefinition-definitions.html#StructureDefinition.base){:target="_blank"}
+* [Name](http://hl7.org/fhir/DSTU2/structuredefinition-definitions.html#StructureDefinition.name){:target="_blank"}
+
+_List of StructureDefinitions_
+
+* [`patient-friendly-display`](http://fhir.cerner.com/millennium/dstu2/medications/medication-order/#custom-extensions): Display string suitable for patient viewing.
+* [`scheduling-location`](http://fhir.cerner.com/millennium/dstu2/scheduling/schedule/#custom-extensions): Reference to the location an appointment being scheduled.
```

Comment:
Just noticed a typo here. 'Reference to the location an appointment being scheduled.' 
Should be updated to: 'Reference to the location **of** an appointment being scheduled.'